src/mds/MDBalancer.h

```diff
@@ -75,6 +76,10 @@ class MDBalancer {
   void handle_mds_failure(mds_rank_t who);
 
   int dump_loads(Formatter *f, int64_t depth = -1) const;
+  int get_rank_mask_bitset(CDir *dir, std::bitset<MAX_MDS>& rank_mask_bitset, bool inherit=true);
```

Comment:
Let's typedef `std::bitset<MAX_MDS>`. Put it in `mdstypes.h`.

Reply:
Defined using typedef in `mdstypes.h`